Directory.Build.props

```diff
@@ -1,34 +1,21 @@
 <Project>
 
-  <PropertyGroup>
-    <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <LangVersion>8.0</LangVersion>
-    <TreatWarningsAsErrors Condition="'$(Configuration)' == 'Release'">true</TreatWarningsAsErrors>
-    <NoWarn>$(NoWarn);CS1591;CA1812;SA1309;SA1600;SA1633;SA1200;SA1202;SA1201;CA1016;CA1303</NoWarn>
-  </PropertyGroup>
+	<PropertyGroup>
+		<GenerateDocumentationFile>true</GenerateDocumentationFile>
+		<LangVersion>latest</LangVersion>
+		<TreatWarningsAsErrors Condition="'$(Configuration)' == 'Release'">true</TreatWarningsAsErrors>
+		<NoWarn>$(NoWarn);CS1591;CA1812;SA1309;SA1600;SA1633;SA1200;SA1202;SA1201;CA1016;CA1303</NoWarn>
+	</PropertyGroup>
 
-  <PropertyGroup>
-    <Description>A client library for KMD Logic Gateway Automation</Description>
-    <VersionPrefix>0.1.0</VersionPrefix>
-    <Authors>KMD</Authors>
-    <PackageTags>kmd;logic;Gateway</PackageTags>
-    <PackageIconUrl>https://console.kmdlogic.io/favicon-32x32.png</PackageIconUrl>
-    <PackageProjectUrl>https://github.com/kmdlogic/kmd-logic-gateway-automation</PackageProjectUrl>
-    <PackageLicenseExpression>MIT</PackageLicenseExpression>
-    <RepositoryUrl>https://github.com/kmdlogic/kmd-logic-gateway-automation</RepositoryUrl>
-    <RepositoryType>git</RepositoryType>
-    <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
-  </PropertyGroup>
```

Comment:
Why this has been removed

Reply:
As far as I know, we only expose dotnet tool as NuGet package, not library or even sample. So IMO we do not need this.

Reply:
We expose library NuGet also, we give more option to consumer, he can use automation NuGet or Dotnet tool.

Reply:
Ooooh, I didn't know. So for sure it must be included.